src/main/kotlin/uk/gov/dluhc/printapi/config/EmailContentProperties.kt

```diff
@@ -15,7 +15,8 @@ class EmailContentConfiguration(
     portalBaseUrl: URI,
     vacContextRoot: String,
     aedContextRoot: String,
-    val certificateReturned: EmailContentProperties
+    val certificateReturned: EmailContentProperties,
+    val certificateFailedToPrint: EmailContentProperties,
```

Comment:
Sorry I didn't understand the need of adding 2 content properties of same type? Can't this be captured in 1 val?

Reply:
This class is called as `EmailContentProperties` whereas it's actually EmailContentConfiguration. Could you please check this for me?

Reply:
This confused me initially - I think the file just needs to be renamed to `EmailContentConfiguration`.

Reply:
Renamed file to EmailContentConfiguration.kt